examples/msw/README.md

```diff
@@ -0,0 +1,25 @@
+# Remix + MSW
+
+This example demonstrates [MSW's][msw] usage with Remix to mock external APIs called from the Remix server during development.
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/msw)
+
+## Example
+
+If the external APIs are metered and charged for the number of API calls made, it's pretty easy to burst through the API quota during development. Also, we need an active network during development. For some, this may be an issue. So instead, we can mock the external API using MSW, which intercepts the API calls from the server and returns a mocked response.
```

Comment:
Those are some great points! 

When I was researching the applications of API mocking, having the ability to develop against a non-existing backend was one of the most common ones. This also applies to situations when you have a backend but it's not in the state that you need (you can augment responses with mocking as well). 

API mocking is also powerful for debugging, as you can reliably reproduce any HTTP call by mocking it. 

Perhaps, consider some of these points. You can read more about them [here](https://mswjs.io/docs/#when-to-mock-api).